tests/run_full.py

```diff
@@ -0,0 +1,51 @@
+# /usr/bin/env python3
```

Comment:
Write `#!/usr/bin/env python3` (that was a bug) and make file executable (`chmod +x tests/run_full.py`).